### PR TITLE
Added malware self tests

### PR DESCRIPTION
**Pull request by MrJnrman**
This PR adds self tests for the new malware test type being added to the QIT

### CODE REVIEW

**Comment by zhongruige** [comment]
Minor nit, but we can also add the test results to the top of the README with the others https://github.com/woocommerce/qit-cli#readme

**Comment by MrJnrman** [comment]
Thanks for calling that out @zhongruige. Added the workflow and badge to this pr

File: .github/workflows/qit-self-test-malware.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    - cron: '0 23 * * *'
+  # Manually
+  workflow_dispatch:
+  # On push to trunk
+  push:
+    branches:
+      - trunk
+
+jobs:
+  phpcompatibility_tests:
```
Comment (Luc45) [inline .github/workflows/qit-self-test-malware.yml]:
```suggestion
  malware_tests:
```

**Review by Luc45 — APPROVED** [review]
Looking good, thanks!

**Pull request merged**